ui/core/src/ui-core/style/itwinui-overrides.scss

```diff
@@ -46,14 +46,16 @@
 
 @mixin small-itwinui-expandable-block {
 
+  $small-space: 5px;
+
   .iui-expandable-block {
     > .iui-header {
-      padding: $iui-xs $iui-xs !important;
+      padding: $small-space $small-space !important;
 
       > .iui-icon {
         width: $uicore-icons-x-small !important;
         height: $uicore-icons-x-small !important;
-        margin-top: $iui-xs !important;
+        margin-top: $small-space !important;

```

Comment:
Vertical alignment looks better now, but the chevron still looks slightly off horizontally.

![firefox_CYbgiG58UI](https://user-images.githubusercontent.com/70327485/132689241-59e2198e-ec30-4f3a-b9e4-abaf48078c4f.png)

Reply:
Does it look different than these: https://itwin.github.io/iTwinUI-react/?path=/docs/core-expandableblock--basic ? Please suggest a satisfactory margin-left.

Reply:
iTwinUI-react ExpandableBlock has 12px padding on the header and 12px right margin on the chevron. To achieve similar balance at our current 5px header padding, I'd add 3px to chevron's left margin and set its right margin to 8px.

Reply:
I pushed a fix

Reply:
Thanks